webextensions/api/scripting.json

```diff
@@ -0,0 +1,351 @@
+{
+  "webextensions": {
+    "api": {
+      "scripting": {
+        "InjectionTarget": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/InjectionTarget",
+            "support": {
+              "chrome": {
+                "version_added": "88"
+              },
+              "edge": {
+                "version_added": "88"
+              },
+              "firefox": {
+                "version_added": "101"
+              },
+              "firefox_android": {
+                "version_added": "101"
+              },
+              "opera": {
+                "version_added": "74"
+              },
+              "safari": {
+                "version_added": "15.4",
+                "notes": "Supported in Manifest V2 and V3 as a parameter of <code>executeScript</code>, <code>insertCSS</code>, and <code>removeCSS</code>."
+              },
+              "safari_ios": {
+                "version_added": "15.4",
+                "notes": "Supported in Manifest V2 and V3 as a parameter of <code>executeScript</code>, <code>insertCSS</code>, and <code>removeCSS</code>."
+              }
+            }
+          }
+        },
+        "RegisteredContentScript": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/RegisteredContentScript",
+            "support": {
+              "chrome": {
+                "version_added": "88"
+              },
+              "edge": {
+                "version_added": "88"
+              },
+              "firefox": {
+                "version_added": "101"
+              },
+              "firefox_android": {
+                "version_added": "101"
+              },
+              "opera": {
+                "version_added": "74"
+              },
+              "safari": {
+                "version_added": null
```

Comment:
We know that Safari doesn't support these parts of the API, so use `"version_added": false` instead of `null`.

(this applies to all entries; if you have doubts about the level of support, tag me)

Reply:
I assume this means that every instance where we have "null" for Safari support it should be replaced with `false`. Done.